src/main.rs

```diff
@@ -52,18 +41,11 @@ async fn main() -> Result<()> {
     Ok(())
 }
 
-fn extract_signer_from_keystore(
-    keystore: &GenericKeyStore,
-) -> Result<subxt_signer::sr25519::Keypair> {
-    let sr25519_pubkey = keystore
-        .iter_sr25519()
-        .next()
-        .ok_or_eyre("No sr25519 keys found in the keystore")?;
-    let sr25519_secret = keystore
-        .expose_sr25519_secret(&sr25519_pubkey)?
-        .ok_or_eyre("No sr25519 secret found in the keystore")?;
-
-    let mut seed = [0u8; 32];
-    seed.copy_from_slice(&sr25519_secret.to_bytes()[0..32]);
-    subxt_signer::sr25519::Keypair::from_secret_key(seed).map_err(Into::into)
+fn init_logger() {
```

Comment:
This seems like something that should be in the sdk?

Reply:
This is related: https://github.com/webb-tools/gadget/pull/285#discussion_r1769076592

I don't think the sdk should have logger initialization, that means we'd be making assumptions about people's logging setup. Having it in the template just makes it a suggestion, and users can swap it out for whatever.